iohub/ngff/nodes.py

```diff
@@ -1525,6 +1555,11 @@ def from_positions(
 
         >>> new_plate = Plate.from_positions("combined.zarr", fovs)
         """
+        # TODO: remove when zarr-python adds back `copy_store`
+        raise NotImplementedError(
```

Comment:
Blocked by upstream regression https://github.com/zarr-developers/zarr-python/issues/2407

Reply:
Tracked in https://github.com/czbiohub-sf/iohub/issues/300.